.github/workflows/deploy-website.yml

```
@@ -87,3 +97,30 @@ jobs:
+    steps:
+      - name: Checkout
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
+
+      - name: Configure bot
+        uses: ./.github/actions/configure-bot
+        with:
+          app-id: ${{ secrets.S2C_BOT_APP_ID }}
+
+      - name: Commit LAST_MODIFIED update
+        env:
+          LAST_MODIFIED: ${{ needs.export.outputs.last-modified }}
```

actionlint (1.7.11)

[error] property "export" is not defined in object type {deploy: {outputs: {}; result: string}}

(expression)